pkg/pillar/types/dpc.go

```diff
@@ -1079,7 +1084,7 @@ type NetworkXObjectConfig struct {
 	UUID                 uuid.UUID
 	Type                 NetworkType
 	Dhcp                 DhcpType // If DhcpTypeStatic or DhcpTypeClient use below
-	Subnet               net.IPNet
+	Subnet               *net.IPNet
```

Comment:
Given that you renamed Subnet to ConfiguredSubnet elsewhere, how about adding an explicit comment here that Subnet, Gateway, etc are the statically configured values when DHCP is not used?

Reply:
Done, comment added.